model/countries/AU/AU-parsing-rules.yaml

```diff
@@ -0,0 +1,304 @@
+regex_definitions:
+  kBuildingValueRe:  # Regex for type "building"
+    regex_concat:
+      parts:
+      - regex_fragment: \d+  # House number
+      # Covering cases - housnumber with alphabetic suffix, housnumber range.
+      - regex_fragment: '(?:'
+      - regex_fragment: \s*[[:alpha:]]\b # Optional suffix for building ids with a letter suffix, e.g. 12a.
+      - regex_fragment: '|'
+      - regex_fragment: \s*?[-]\s*\d+ # Housnumber range (10-17)
+      - regex_fragment: ')?'
+      wrap_non_capture: false
+
+  kUnitTypeLiteralRe:  # Regex for "unit-type"
+    regex_fragment: (?:apt\.?|apartment|se\.?|ste\.?|suite|unit|u\.?|room|rm\.?|shop|po box|ms\.?|mail shop|mb\.?|marine berth|vlla\.?|villa|we\.?|warehouse|office|off\.?|fy\.?|factory|sl\.?|stall|site|shed)
+
+  kUnitNameValueRe:  # Regex for "unit-name"
+    regex_fragment: (?:\d+\w?\b|\w\b)
+
+  kFloorLiteralRe:  # Regex for floor literal
+    regex_fragment: (?:level|l\.?|floor|fl\.?|basement|b\.?|ground floor|g\.?|lower ground floor|lg\.?|upper ground floor|ug\.?|mezzanine|m\.?)
+
+capture_definitions:
+  StreetAddressDecomposition:
+    capture:
+      output: street-address-alternative-1
+      parts:
+      - capture_reference: ParseInBuildingLocation
+      # Matches string of spaces (2 or more) or if in-building location not presented, matches empty space.
+      - separator: {regex_fragment: '(?:[^\S\r\n]{2,}|.*?)'} 
```

Comment:
How do you feel about just using `regex_reference: kWhitespaceSeparator`?

My thought is:
1) do all users in Australia know that they should use two spaces instead of one?
2) does it make a difference to use given that `ParseInBuildingLocation` relies on expressions with mandatory prefixes?
3) `.*?` would consume a single space as well, right?